app/spdb/scripts/spdb

```diff
@@ -159,7 +159,7 @@ start ()
 
 	    # Attempt to start.
 	    printf "Starting ODB ${SPDB_VERSION_STR}"
-	    "${SPDB_EXEC}" 2>&1 > "${SPDB_DIR}/sysout.txt" &
+	    /sbin/runuser software -s /bin/bash -c "nohup ${SPDB_EXEC} 2>&1 > ${SPDB_DIR}/sysout.txt &"
```

Comment:
Is it possible to use `sudo -u software` and test it that way?

Reply:
I was reading up on it and apparently, in some cases, `sudo` asks for a password (even when run as root), which obviously can't be provided in an init.d script, although that was one solution that I did see proposed. The advantage of `runuser` is that it never asks for a password as it can only be run as `root`.

Reply:
ok, sounds good. i guess this is difficult to test in so perhaps you can put the new script in gsnodb test and restart to verify